tests/test_jsonschema/test_json_schema_string_format.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from typing import Annotated
```

Comment:
It should be replaced with `from typing_extensions import Annotated` to support Python 3.8